### ISSUE TITLE
I really want to know where the pruning you did before upsampling was

### ISSUE THREAD

**Issue by yueshizaishengqi** [Robertwyq/PanoOcc#15]
You said you prune the query to a sparse feature, but I didn't see where you used prune
![1718960957739](https://github.com/Robertwyq/PanoOcc/assets/165989984/689f8b61-0653-44e3-a91a-29859fdb4af9)
![1718961034763](https://github.com/Robertwyq/PanoOcc/assets/165989984/64d2de68-b063-46d3-b7a7-25d5971d9387)
all the things in `self.upsampling` is `nn.ConvTranspose3d`, `nn.BatchNorm3d`, `nn.ReLU`, where you used prune?

If you prune the query after upsampling, I want to know why in your picture, the order is prune -> unsampling -> prune?
![1718961911298](https://github.com/Robertwyq/PanoOcc/assets/165989984/e27c0fe5-2d8f-4f4a-9532-dafd330ad3e2)
Assuming it is indeed so, you say you employ 3 layers of 3D deconvolutions, so the order is prune -> unsampling -> prune -> unsampling -> prune -> unsampling -> prune, prunning four times, but you only provide three of the keeping ratios: 0.2, 0.5, 0.5, how can you explain this part?
![1718961882774](https://github.com/Robertwyq/PanoOcc/assets/165989984/fd7eee3f-4c92-40a1-84f0-fb44c696f424)
